### PR TITLE
Landing page update

### PR DESCRIPTION
**Pull request by LanesGood**
Opening this as a draft until lad content is finalized.

### CODE REVIEW

**Review by j08lue — COMMENTED** [review]
The text could use a more thorough rework, replacing some of the very implementation-specific descriptions (e.g. FastAPI) with more function-oriented lingo (e.g. REST API service).

Not what I have done in my suggestions here - I just swapped out a few words I stumbled over.

File: docs/src/overrides/home.html [hunk 1]
```diff
@@ -3,311 +3,311 @@
...
+                <header class="md-typeset text-centered">
+                    <h5 id="trusted-by" class="supheading">Trusted By</h5>
+                </header>
+                <div class="feature-grid logo-grid" style="--cols: 3;">
+                    <figure>
+                        <img src="../img/nasa_logo.svg" alt="NASA logo" loading="lazy" height="88px">
+                    </figure>
+                    <figure>
+                        <img src="../img/msft_logo.svg" alt="Microsoft Logo" loading="lazy" height="64px">
+                    </figure>
```
Comment (vincentsarago) [inline docs/src/overrides/home.html:274]: We should remove Microsoft!!! They do use a similar approach (will the base services) but it's not eoapi!

**Comment by zacharyDez** [comment]
> The text could use a more thorough rework, replacing some of the very implementation-specific descriptions (e.g. FastAPI) with more function-oriented lingo (e.g. REST API service).

@j08lue Lane focused on structure; we can change the content as needed. I'll look into how we can include a functional description. Should it be in the bold gray subtitle like below:

image

The intent of the first bold title was to be functionaly driven at a higher level - Catalog your data, make it discoverable, visualize it. The newer grey subtitles might provide what we deem a functional description for more technical audiences? It seems safe to assume a developer interest in EO/geo understands what tiling means.

**Comment by zacharyDez** [comment]
Regarding logos for the trusted by section, @j08lue @aboydnw  @kcarini, can you let me know here or via Slack the steps we need to get approval or if we already have it?

File: docs/src/overrides/home.html (hunk 1 above)
Comment (zacharyDez) [reply docs/src/overrides/home.html]: Removed.

File: docs/src/overrides/home.html [hunk 2]
```diff
@@ -3,311 +3,308 @@
...
+                                <p>
+                                    <a href="https://github.com/stac-utils/pgstac">stac-utils/pgstac</a>
+                            </div>
+                        </article>
+                    </li>
+                    <article class="feature-item">
+                        <img src="../img/stacfastapi_logo.png" alt="stac-fastapi logo" loading="lazy" />
+                        <div class="feature-item__description md-typeset">
+                            <h3>
+                                Make it Discoverable
```
[the remark this answers is not in the document]
Comment (zacharyDez) [reply docs/src/overrides/home.html]: good distinction! Appreciate your suggestion @edkeeble. Not pedant at all, please bring more!

**Comment by aboydnw** [comment]
@zacharyDez I can ask on getting approval for the NASA logo, we might just have to submit a form linking to how we want to use it

**Comment by zacharyDez** [comment]
@j08lue, do the changes match your expectations on re-wording the content? Any other feedback on it?

**Comment by zacharyDez** [comment]
@kcarini @olafveerman @ianschuler @vincentsarago @sharkinsspatial @j08lue; could you give a second look when you can? I want to keep the ball rolling here and merge when we get approval for the logos.

**Comment by zacharyDez** [comment]
Thanks to Vincent 🙏 , the new version is deployed here: https://eoapi-dev1.surge.sh/

**Review by vincentsarago — APPROVED** [review]

**Review by vincentsarago — APPROVED** [review]

**Pull request merged**